third_party/opa/data-and-policies/user_policies.rego

```diff
@@ -0,0 +1,75 @@
+package user_policies
+
+import data.data_policies as dp
+
+#Example of data policies that use "data_policies" package to create easily data policies that deny access or transform the data accordingly
+
```

Comment:
Not urgent, but I would suggest modeling the policies detailed in the customer story we are working on as well